src/desktop/models/page.coffee

```diff
@@ -11,7 +11,5 @@ module.exports = class Page extends Backbone.Model
   urlRoot: "#{sd.API_URL}/api/v1/page"
 
   sanitizedHtml: (attr) ->
-    clean = sanitizeHtml(@get(attr), {
-      allowedTags: sanitizeHtml.defaults.allowedTags.concat([ 'h1', 'h2', 'summary', 'details' ])
-    })
+    clean = insane(@get(attr))
```

Comment:
@alloy can you verify (with screenshot, etc.) that we're still properly rendering pages that show markdown (like https://staging.artsy.net/conditions-of-sale)?

Also, does `insane` have its own definition of what tags are allowed? Does it include those above?

Reply:
Good call, I was relying on the tests to convey that this was still the case.

<img width="616" alt="Screenshot 2019-10-10 at 19 11 19" src="https://user-images.githubusercontent.com/2320/66590781-d18d6280-eb91-11e9-807b-8013192cbd09.png">

Reply:
As for the allowed tags, it has the same as sanitized-html _plus_ the ones you previously added manually https://github.com/bevacqua/insane/blob/v2.6.1/defaults.js#L12-L15

Reply:
Seems correct!

I remember looking at `insane` for this but I can't remember why I decided to go with `sanitize-html`. Should have done a better job documenting the alternatives.

Reply:
The dependency RFC process helps with this!